### PR TITLE
Renamed cluster_id attribute from message object to integration_id

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
also modified to log the mesages using log_utils function

tendrl-bug-id: Tendrl/commons#814

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/commons/flows/__init__.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from tendrl.commons.event import Event
```
Comment (shtripat) [inline tendrl/commons/flows/__init__.py]: Do you need `Event`?

File: tendrl/commons/flows/create_cluster/ceph_help.py [hunk 2]
```diff
@@ -4,67 +4,53 @@
```
Comment (r0h4n) [inline tendrl/commons/flows/create_cluster/ceph_help.py]: remove this file and any other ceph file

File: tendrl/commons/logger.py [hunk 3]
```diff
@@ -54,16 +54,16 @@ def push_operation(self):
...
+                self.message.integration_id != ""):
                 NS.node_agent.objects.ClusterMessage(
                     message_id=self.message.message_id,
                     timestamp=self.message.timestamp,
                     priority=self.message.priority,
                     publisher=self.message.publisher,
                     node_id=self.message.node_id,
                     payload=self.message.payload,
-                    cluster_id=self.message.cluster_id,
+                    integration_id=self.message.integration_id,
```
Comment (r0h4n) [inline tendrl/commons/logger.py]: any impact on UI/API?

File: tendrl/commons/objects/cluster/atoms/create/__init__.py [hunk 4]
```diff
@@ -5,9 +5,9 @@
 from tendrl.commons.flows.create_cluster.gluster_help import create_gluster
```
Comment (r0h4n) [inline tendrl/commons/objects/cluster/atoms/create/__init__.py]: we no longer support create cluster, please remove this file and supporting files

Comment (GowthamShanmugam) [reply tendrl/commons/objects/cluster/atoms/create/__init__.py]: i will remove it

File: tendrl/commons/logger.py (hunk 3 above)
Comment (GowthamShanmugam) [reply tendrl/commons/logger.py]: no, it won't affect UI/API

File: tendrl/commons/flows/create_cluster/ceph_help.py (hunk 2 above)
Comment (GowthamShanmugam) [reply tendrl/commons/flows/create_cluster/ceph_help.py]: i will remove it

File: tendrl/commons/flows/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/commons/flows/__init__.py]: yes, because ExceptionMessage class is using Event, for that we can't use log_util

Comment (GowthamShanmugam) [reply tendrl/commons/flows/__init__.py]: in this file ExceptionMessage is used so i am keeping Event

Comment (shtripat) [reply tendrl/commons/flows/__init__.py]: ok

**Comment by r0h4n** [comment]
@GowthamShanmugam please fix failing tests due to ceph removals

**Pull request merged**